.gitignore

```diff
@@ -105,3 +105,4 @@ ENV/
 /dataset/
 /config/
 /tmp/
+/.idea/
```

Comment:
@ananno Can you add to your local `~/.gitignore` ?

Reply:
OK